scipy/stats/stats.py

```diff
@@ -2789,10 +2789,15 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
         A function that will return the central value. The default is to use
         np.median. Any user defined function used will need to have the function
         signature ``func(arr, axis)``.
-    scale : int, optional
-        The scaling factor applied to the MAD. The default scale (1.4826)
-        ensures consistency with the standard deviation for normally distributed
-        data.
+    scale : scalar or str, optional
+        The numerical value of scale will be divided out of the final
```

Comment:
This docstring does not match the code. You have to decide if you are going to keep the old behavior and multiply the result by `scale` or adopt a new behavior and divide by `scale` to match `iqr`. Right now the docstring says divide but the code does a multiplication. Similarly, the docstring says that `'normal'` equals 1.4826, but in fact the code produces scale=1/1.4826=0.6745.

Reply:
@u55 yes, one was multiplying the other dividing. I'm working on fixing the tests now (the doctests will need to change as well), thanks for catching that.

Reply:
Looked at this closer over the last couple days. So the point here with the scaling factor is to produce an asymptotically unbiased estimator of the standard deviation, assuming all observations are draws from a normal distribution with mean 0. To ensure that that is the case the MAD needs to either get multiplied by 1.4826 (reciprocal of the inverse cdf of std normal at 0.75) or what is equivalent, dividing by inverse cdf of std normal at 0.75. 
As we want to have alignment/consistency with `iqr` I've adjusted here to match `iqr`